cmd/minishift/cmd/start_preflight.go

```diff
@@ -414,3 +423,29 @@ func isMounted(driver drivers.Driver, mountpoint string) (bool, error) {
 
 	return true, nil
 }
+
+// checkIsoUrl checks the Iso url and returns true if the iso file exists
+func checkIsoUrl() bool {
+	isoUrl := viper.GetString(configCmd.ISOUrl.Name)
+
+	if isoUrl == constants.B2dIsoAlias || isoUrl == constants.CentOsIsoAlias || isoUrl == constants.MinikubeIsoAlias {
+		return true
+	}
+
+	if runtime.GOOS == "windows" {
+		match, _ := regexp.MatchString("^file://[a-z]:/.+", isoUrl)
+		if match != true {
+			return false
+		}
+	}
```

Comment:
Line 441-444 need to be inside else block of above if condition since it checks Unix file path. Hence, I was getting preflight failure error even in passing valid iso url path.